internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -875,6 +876,42 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 				Computed:  true,
 				Sensitive: true,
 			},
+
+			"http_proxy_config": {
+				Type:     pluginsdk.TypeList,
+				Computed: true,
+				Optional: true,
+
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"http_proxy": {
+							Type:     pluginsdk.TypeString,
+							Computed: true,
+							Optional: true,
+						},
+						"https_proxy": {
+							Type:     pluginsdk.TypeString,
+							Computed: true,
+							Optional: true,
+						},
+						"no_proxy": {
+							Type:     pluginsdk.TypeList,
+							Computed: true,
+							Optional: true,
```

Comment:
Based on the docs, `no_proxy` cannot be changed, thus `ForceNew`
```suggestion
							Optional: true,
							ForceNew: true,
```